From 2b0d2225caaad09a5b363a8879a347f28eed38ad Mon Sep 17 00:00:00 2001
From: Dale MacDonald <lawrence.macdonald@colorado.edu>
Date: Sun, 2 Mar 2025 14:15:20 -0800
Subject: [PATCH] patch

---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7b08c49..d089b7c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,9 +1,11 @@
 
-SUBDIRS =  misc-progs misc-modules \
+# SUBDIRS =  misc-progs misc-modules \
            skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
 	   pci usb\
 	   sbull snull short
 
+SUBDIRS = misc-modules scull
+
 all: subdirs
 
 subdirs:
-- 
2.34.1

